src/Aggregate/AggregateDescription.php

```diff
@@ -107,6 +161,11 @@ abstract protected static function commandServiceMapping(): array;
      */
     abstract protected static function commandPreProcessors(): array;
 
+    /**
+     * @return array<int, class-string>
+     */
+    abstract protected static function projectorsAggregateMapping(): array;
```

Comment:
I think this is not really a sort of mapping, but rather a list of all the projectors?

Reply:
True, the function has changed

Reply:
Then, we should rename the function too?